test/e2e/tests/ng-route-promise.spec.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+describe('ngRoute promises', function() {
+  beforeEach(function() {
+    loadFixture('ng-route-promise');
+  });
+
+  it('should wait for route promises', function() {
+    expect(element.all(by.tagName('li')).count()).toBe(5);
+  });
+
+  it('should time out if the promise takes long enough', function() {
+    // Don't try this at home kids, I'm a protractor dev
+    browser.manage().timeouts().setScriptTimeout(1500);
+    browser.waitForAngular().then(function() {
+      fail('waitForAngular() should have timed out, but didn\'t');
+    }, function(error) {
+      var errorRe = new RegExp(
+          'Timed out waiting for asynchronous Angular tasks to finish after ' +
+          '(?:1\\.5 seconds|15\\d\\d ?ms)\\.', '');
```

Comment:
I know you took this from my PR, but this RegExp is testing more than is necessary (and might give false negatives after a future update). I think it suffixes to have:

```js
expect(error.message).toContain('Timed out waiting for asynchronous Angular tasks to finish');
```